packages/env/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### Breaking Changes
+-   Removed the `WP_PHPUNIT__TESTS_CONFIG` environment variable from the `phpunit` container. **This removes automatic support for `wp-phpunit/wp-phpunit`**.
```

Comment:
Could you add a line saying this is the composer package? I get confused about that sometimes. It might also be worth explicitly saying that this is replaced by the enhancement below.